packages/next/src/views/CreateFirstUser/index.tsx

```diff
@@ -61,6 +62,25 @@ export const CreateFirstUserView: React.FC<AdminViewProps> = async ({ initPageRe
     fieldSchema: fields,
     i18n,
     parentPath: userSlug,
+  }).map((field) => {
```

Comment:
This is still technically a second loop, but tbh its probably acceptable at this point.

Reply:
Alternatively you could probably update `mapFields` to accept these new field types

Reply:
I wouldn't mind the second one but it might go deeper than that function to update the `Field` type too, these fields aren't used anywhere else so I think it's safe to merge it like this for now